docs/release-notes/v12.0.1.md

```diff
@@ -0,0 +1,135 @@
+---
+id: v12.0.1
+title: Version 12.0.1 Release Notes
+sidebar_label: v12.0.1
+image: "img/release-note-link-preview.png"
+---
+
+_December 1st, 2023_
+
+
+:::info
+Mia-Platform Console v12.0.1 is **now in Preview!**
+
+Console SaaS users can try out v12.0.1 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## Console
+
+### Improvements
+
+#### MongoDB CRUD section now manages Fast Data `sv-error` collection
+
+Fast Data collection `sv-error` is now visible in the MongoDB CRUD section.
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* a bug has been fixed that causes the user to lose local changes in case of exiting the Design Area after a successful merge process;
```

Comment:
```suggestion
* a bug has been fixed that causes the user to lose local changes in case of exiting the Design Area after a successful merge process
```